src/main/java/uk/gov/hmcts/reform/lrdapi/controllers/WelcomeController.java

```diff
@@ -1,9 +1,12 @@
 
 package uk.gov.hmcts.reform.lrdapi.controllers;
 
-import io.swagger.annotations.ApiOperation;
-import io.swagger.annotations.ApiResponse;
-import io.swagger.annotations.ApiResponses;
+//import io.swagger.annotations.ApiOperation;
```

Comment:
Is this commented line required or can we remove it

Reply:
Removed Thanks @manukundloo-hmcts